packages/frontend/src/lib/__tests__/getApiBaseUrl.test.ts

```
@@ -0,0 +1,13 @@
+import { getApiBaseUrl } from '../getApiBaseUrl';
+
+describe('getApiBaseUrl', () => {
+  it('retourne la variable VITE_API_URL si process.env est défini', () => {
+    process.env.VITE_API_URL = 'http://test-from-process:1234';
+    expect(getApiBaseUrl()).toBe('http://test-from-process:1234');
+    delete process.env.VITE_API_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)